docpages/example_programs/interactions_and_components/commandhandler.md

```diff
@@ -11,7 +10,7 @@ Note that because the dpp::commandhandler::add_command method accepts a `std::fu
 at a simple lambda (as shown in this example), a function pointer, or an instantiated class method of an object. This is extremely flexible
 and allows you to decide how and where commands should be routed, either to an object oriented system or to a lambda based system.
 
-\warning As of August 30th, 2022, you are advised to only be using slash commands, not messages for commands. To prevent the command handler from handling commands with messages, you should only use the "/" prefix. If you wish to still use messages for commands, this tutorial will still cover it but, again, it is discouraged by Discord.
+\warning As of August 23rd, 2023, you are advised to only be using slash commands, not messages for commands. To prevent the command handler from handling commands with messages, you should only use the "/" prefix. If you wish to still use messages for commands, this tutorial will still cover it but, again, it is discouraged by Discord.
```

Comment:
This is wrong. https://support-dev.discord.com/hc/en-us/articles/6025578854295-Why-We-Moved-to-Slash-Commands.

I made this edit recently and changed it for a reason 🙃

Reply:
Ah, I see, I'll think it there, as it's not clear that it isn't just not up to date.